src/Lumi/Components/Table/FilterDropdown.purs

```diff
@@ -106,19 +108,23 @@ filterItem_ = makeStateless filterItemComponent render
                   connectDragSource $ connectDropTarget $
                     element (unsafeCreateDOMComponent "lumi-row")
                       { className: if item.hidden then "" else "active"
-                      , style: css
-                          { padding: "0 8px 0 0"
-                          , alignItems: "center"
-                          , borderTop:
-                              if isOver && (fromMaybe false ((\dragItem -> dragItem.index > index) <$> maybeDragItem))
-                              then "2px solid #0044e4"
-                              else "2px solid transparent"
-                          , borderBottom:
-                              if isOver && (fromMaybe false ((\dragItem -> dragItem.index < index) <$> maybeDragItem))
-                              then "2px solid #0044e4"
-                              else "2px solid transparent"
-                          , opacity: if isDragging then 0.1 else 1.0
-                          }
+                      , style:
+                        let
+                          borderStyle :: (Int -> Int -> Boolean) -> String
+                          borderStyle compare' =
+                            if isOver && (fromMaybe false ((\dragItem -> dragItem.index `compare'` index) <$> maybeDragItem))
```

Comment:
This is the same as
```purescript
if isOver && maybe false (\dragItem -> dragItem.index `compare` index) maybeDragItem
```

Reply:
I'm passing in both `<` and `>` though, or `compare` and `flip compare`

Reply:
I think Arthur just meant that `fromMaybe y (f <$> x)` is (arguably) better written as `maybe y f x`.

Reply:
Ohhhh, got it

Reply:
Yeah, just mentioned that as a suggestion as I personally find the latter clearer to read than the former :)